src/ducks/account.ts

```diff
@@ -69,15 +126,35 @@ const accountSlice = createSlice({
       // sure we have the response data to set isAuthenticated correctly.
       isAuthenticated: !!action.payload,
     }));
-
     builder.addCase(fetchAccountAction.rejected, (state, action) => ({
       ...state,
       data: null,
       status: ActionStatus.ERROR,
       errorString: action.payload?.errorString,
     }));
+
+    // TODO: figure out why it breaks for BigNumber amount
```

Comment:
What does "break" mean in this context? And I'm guessing it's not important enough or doesn't effect the UI such that we should fix it right away?

Reply:
It is this guy: https://github.com/stellar/account-viewer-next/issues/60. We spent a little bit of time on it and couldn't find a fix. It looks like it wants you to always provide values for everything in state. If we spread and update just one thing, it complains. 🤷‍♀️ It's on the list for this sprint. 🤞